tasks/github_tasks.py

```diff
@@ -68,6 +68,7 @@ def trigger_macos(
     retry_download=3,
     retry_interval=10,
     fast_tests=None,
+    test_washer=None,
```

Comment:
nit: you used `False` as the default value in the other task. We might change it to be consistent